src/views/PurchaseOrder.vue

```diff
@@ -112,28 +143,35 @@ export default defineComponent({
           showToast(translate("Something went wrong, Please try again"));
         }
       },
-
       async parseFile(){
         await parseCsv(this.file).then(res => {
           this.content = res;
         })
       },
-      mapFields() {
+      review() {
+        if(!this.orderItemsList.length) this.mapFields();
+        this.store.dispatch('order/updatedOrderList', this.orderItemsList);
+        this.router.push({
+          name:'PurchaseOrderDetail'
+        })
+      },
+      mapFields(event) {
```

Comment:
We could check if mapping fields exist in csv

Reply:
Made the requested change.
Also displayed a toast if any of the fields is missing.